src/libs/actions/Report.ts

```diff
@@ -222,6 +222,13 @@ Onyx.connect({
     },
 });
 
+let reports: OnyxCollection<Report>;
+Onyx.connect({
+    key: ONYXKEYS.COLLECTION.REPORT,
+    waitForCollectionCallback: true,
+    callback: (value) => (reports = value),
+});
+
```

Comment:
I have no issue with this change, but we're also [refactoring report collection here](https://github.com/Expensify/App/pull/44068), so I think we can probably hold this PR for https://github.com/Expensify/App/pull/44068 (it's close to merge). After that, we can remove this block of code and use `ReportConnection.getAllReports()` instead.

Reply:
Nah let's not hold it. We can clean it up after. That's a big PR which might take a while.

Reply:
it's merged already. We can use `ReportConnection.getAllReports()` from now